readme.md

```diff
@@ -468,6 +468,7 @@ Apps that have integration or involve XRPL in some way.
   filter wizard. _Formerly known as ZerpTracker._
 - [xrpayments.co](https://xrpayments.co): Tool to generate payment request QR (with currency conversion) There's a 
   corresponding 📱 App that requires registration here.
+- [XRPhone](https://xrphone.app): Pay merchant invoices over the phone using XRP. [GitHub](https://github.com/jremi/xrphone). 
```

Comment:
Awesome project! :telephone_receiver: :money_with_wings:  

Mind if we change `GitHub` to `Source`?

```suggestion
- [XRPhone](https://xrphone.app): Pay merchant invoices over the phone using XRP. [Source](https://github.com/jremi/xrphone)
```

Reply:
ehh, i'm going to go ahead and do it :)

Reply:
@yyolk thx for updating. 🙏